src/app.py

```diff
@@ -60,48 +61,49 @@
             ])
             ])
         ]),
-        #first plot
-        dbc.Col(html.Iframe(
-            #...
-        )),
-
-        #Second plot
-        dbc.Col(html.Iframe(
-            #...
-        ))
```

Comment:
@zackt113  I believe this was the reason why the sidebar became wider, because one of the two dbc.Col() was deleted